Sources/Datadog/FeaturesIntegration/WebView/DatadogEventBridge.swift

```diff
@@ -0,0 +1,66 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal typealias JSON = [String: Any]
+
+internal protocol WebEventConsumer {
+    func consume(event: JSON, eventType: String)
+}
```

Comment:
How the `eventType` will be used by the consumer? From the bridge implementation, the event type is used to choose which consumer to invoke. So I guess the consumer will only process to event json.

Reply:
you can see it [here in Android](https://github.com/DataDog/dd-sdk-android/blob/feature/hybrid-application/dd-sdk-android/src/main/kotlin/com/datadog/android/webview/internal/WebRumEventConsumer.kt#L50)

RUM events have different `eventType` values although Logs only have `eventType: "log"`